src/sentry/workflow_engine/typings/grouptype.py

```diff
@@ -0,0 +1,20 @@
+from dataclasses import dataclass
+
+from sentry.issues.grouptype import GroupCategory, GroupType
+
+
+# hidden group type, used for issue stream detector
+@dataclass(frozen=True)
+class IssueStreamGroupType(GroupType):
+    type_id = -1
```

Comment:
cc @wedamija -- any recommendations for this type_id here? we really just need a stub issue type for a detector that we're using for data modeling purposes in processing.

i'm a _little_ nervous about using a negative number here, but it seems like an okay placeholder, at least for now.

Reply:
It gets used in `Group` rows and occurrence rows, but if you're not actually making issues of this type you should be fine.

You can also just use a real number though, it probably doesn't make a difference either way, as long as it's defined here.